packages/timeline/src/ReactAxisGantt.ts

```diff
@@ -0,0 +1,244 @@
+import { Axis } from "@hpcc-js/chart";
+import { SVGWidget } from '@hpcc-js/common';
+import { React, Rectangle } from "@hpcc-js/react";
+import { ReactGantt } from "./ReactGantt";
+
+export class ReactAxisGantt extends SVGWidget {
+
+    protected _topAxis: Axis = new Axis("center")
+        .orientation("top")
+        .type("linear")
+        .shrinkToFit("none")
+        .overlapMode("hide")
+        .extend(0)
+        .tickFormat("d")
+        ;
+    protected _gantt: ReactGantt = new ReactGantt("center");
+    protected _bottomAxis: Axis = new Axis("center")
+        .orientation("bottom")
+        .type("linear")
+        .shrinkToFit("none")
+        .overlapMode("hide")
+        .extend(0)
+        .tickFormat("d")
+        ;
+    
+    protected _topAxisElement;
+    protected _contentElement;
+    protected _bottomAxisElement;
+    protected _topRect;
+    protected _contentRect;
+    protected _bottomRect;
+
+    constructor(){
+        super();
+        this._drawStartPos = "origin";
+    }
+
+    private _rangeRenderer: React.FunctionComponent = Rectangle;
+    rangeRenderer(): React.FunctionComponent;
+    rangeRenderer(_: React.FunctionComponent): this;
+    rangeRenderer(_?: React.FunctionComponent): this | React.FunctionComponent {
+        if (!arguments.length) return this._rangeRenderer;
+        this._rangeRenderer = _;
```

Comment:
Don't really need this._rangeRenderer can just proxy to this._gantt.rangeRenderer for both set and get